internal/k8s/controller.go

```diff
@@ -565,22 +565,32 @@ func AddPodEventHandler(numWorkers uint32, c *AviController) cache.ResourceEvent
 			}
 			oldPod := old.(*corev1.Pod)
 			newPod := cur.(*corev1.Pod)
-			if !reflect.DeepEqual(newPod, oldPod) {
-				namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(newPod))
-				key := utils.Pod + "/" + utils.ObjKey(oldPod)
-				if lib.IsNamespaceBlocked(namespace) {
-					utils.AviLog.Debugf("key: %s, msg: Pod Update event: Namespace: %s didn't qualify filter", key, namespace)
-					return
-				}
-				if _, ok := newPod.GetAnnotations()[lib.NPLPodAnnotation]; !ok {
-					utils.AviLog.Warnf("key : %s, msg: 'nodeportlocal.antrea.io' annotation not found, ignoring the pod", key)
-					return
+			key := utils.Pod + "/" + utils.ObjKey(oldPod)
+			namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(newPod))
+			if lib.IsNamespaceBlocked(namespace) {
+				utils.AviLog.Debugf("key: %s, msg: Pod Update event: Namespace: %s didn't qualify filter", key, namespace)
+				return
+			}
+			if _, ok := newPod.GetAnnotations()[lib.NPLPodAnnotation]; !ok {
+				utils.AviLog.Warnf("key : %s, msg: 'nodeportlocal.antrea.io' annotation not found, ignoring the pod", key)
+				return
+			}
+			for _, c := range newPod.Status.ContainerStatuses {
```

Comment:
please use descriptive variable name like `container`. This variable name is same as that of controller variable name `c` being used

Reply:
Have changed it to **container**.